module/VuFind/config/module.config.php

```diff
@@ -478,6 +479,7 @@
             'VuFind\IpAddressUtils' => 'VuFind\Net\IpAddressUtils',
             'VuFind\Logger' => 'VuFind\Log\Logger',
             'VuFind\Mailer' => 'VuFind\Mailer\Mailer',
+            'VuFind\MetadataVocabularyPluginManager' => 'VuFind\MetadataVocabulary\PluginManager',
```

Comment:
This alias is probably unnecessary -- we only have these for legacy compatibility (from the "bad old days" when not all service names matched their class names).